tests/message/unittest_message_id_store.py

```diff
@@ -90,3 +94,47 @@ def test_duplicate_msgid(msgid_store: MessageIdStore) -> None:
         "Message symbol 'warning-symbol' cannot be used for 'W1234' and 'W1235'"
         in str(error.value)
     )
+
+
+def test_exclusivity_of_msg_ids() -> None:
```

Comment:
```suggestion
def test_exclusivity_of_msgids() -> None:
```
This is the convention in ``pylint.messages``.